.github/workflows/dev_deployment.yml

```diff
@@ -0,0 +1,30 @@
+name: Deployment
+
+on:
+    push:
+        branches:
+            - master
```

Comment:
You should first try and run the workflow from your branch before merging to master